test/e2e/common.ts

```diff
@@ -97,7 +97,8 @@ export const withConductor =
   };
 
 export const installAppAndDna = async (
-  adminPort: number
+  adminPort: number,
+  nonExpiringToken?: boolean
```

Comment:
Would it make sense to call this parameter `singleUseToken` to stay close to the conductor API naming?

Reply:
And then have it be true by default? We can do that yes. It's single-use and expiring...I could also make two parameters for expiry and single use.

Reply:
I changed it to take two separate parameters now to be aligned with the conductor API naming: https://github.com/holochain/holochain-client-js/pull/279/commits/d6a7db7113ba2f14a688045a36866e0a83fbf9de

Reply:
That works yes, just expose the API with defaults. I was thinking that single use could have a timeout and when it's false then it could have no expiry. I wouldn't be opposed to the client exposing the API that way because those are the two expected configurations really

Reply:
Though it's probably useful to let people just have access to these